source/jquery.flot.touch.js

```diff
@@ -5,9 +5,7 @@
     'use strict';
 
     var options = {
-        pan: {
-            enableTouch: false
-        }
+        propagateOriginTouch: false
```

Comment:
Instead of a new property to control the propagation of the event, why shouldn't we just use the 'enableTouch' states of the pan and zoom options? See below for the comment dealing with how we do this.